keras/engine/training.py

```diff
@@ -1442,57 +1442,59 @@ def fit(self, x=None,
         """Trains the model for a fixed number of epochs (iterations on a dataset).
 
         # Arguments
-            x: Numpy array of training data,
-                or list of Numpy arrays if the model has multiple inputs.
-                If all inputs in the model are named,
-                you can also pass a dictionary
-                mapping input names to Numpy arrays.
-                Can be `None` (default) if feeding from framework-native tensors.
-            y: Numpy array of target data,
-                or list of Numpy arrays if the model has multiple outputs.
-                If all outputs in the model are named,
-                you can also pass a dictionary
-                mapping output names to Numpy arrays.
-                Can be `None` (default) if feeding from framework-native tensors.
+            x: Numpy array of training data, or 1-element list of Numpy arrays.
+                If the input in the model is named, you can also pass a
+                dictionary mapping input name to Numpy array. Can be
+                `None` (default) if feeding from framework-native tensors.
+            y: Numpy array of target (label) data, or 1-element list of Numpy
+                arrays. If the output in the model is named, you can also pass
+                a dictionary mapping output name to Numpy array. Can be
+                `None` (default) if feeding from framework-native tensors.
             batch_size: Integer or `None`.
                 Number of samples per gradient update.
                 If unspecified, it will default to 32.
-            epochs: Integer, the number of times to iterate
-                over the training data arrays.
+            epochs: Integer. Number of epochs to train the model.
+                Note that in conjunction with initial_epoch, the parameter
+                epochs is to be understood as "final epoch". The model is
+                not trained for a number of steps given by epochs, but
+                until the epoch epochs is reached.
             verbose: 0, 1, or 2. Verbosity mode.
-                0 = silent, 1 = verbose, 2 = one log line per epoch.
-            callbacks: List of callbacks to be called during training.
+                0 = silent, 1 = progress bar, 2 = one line per epoch.
+            callbacks: List of `keras.callbacks.Callback` instances.
```

Comment:
Insert markdown link to the callbacks page (like we do for layer activations, regularizers)

Reply:
Couldn't figure out how it looks like for sure. There is something like it 2 lines down.